src/components/ContentCard/ContentCard.stories.tsx

```diff
@@ -59,7 +59,7 @@ export const WithTag = {
   name: 'With tag',
   args: {
     ...Playground.args,
-    tag: <Tag variant="">Tag text</Tag>
+    tag: <Tag>Tag text</Tag>
```

Comment:
so sorry i missed this in the last round, but can we remove 'text' to align with dummy content for tag